ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -514,3 +524,11 @@ extension PillButtonBar: UIPointerInteractionDelegate {
         return UIPointerStyle(effect: pointerEffect, shape: nil)
     }
 }
+
+// MARK: PillButtonBarDelegate
+
+@objc(MSFPillButtonBarDelegate)
```

Comment:
Nit: why is this moved? I generally see delegate APIs specified before their control, though maybe that's my crusty old AppKit-addled brain thinking.

Reply:
The thinking was that it's internal, not public, so it should be lower. Should I move it back up?

Reply:
Ah, off by one, public, not open.